infrastructure/lambda/api.yaml

```diff
@@ -195,22 +218,15 @@ paths:
         description: 'A valid access_token (e.g.: authorization: Bearer access-token-value).'
         schema:
           type: string
-    post:
-      summary: Post request using a valid access_token
-      description: Accepts form encoded param consisting of the access_token and subject
-
+    get:
+      summary: GET request using a valid access token
```

Comment:
IPV Core Stub uses the `UserInfoRequest` to build a request to /credential/issue. If we use this class, then we need to use a GET in order to set the token in the `Authorization:Bearer xxxx` header. If we use a POST from this class, the token gets set in the req body